lib/ui/painting.dart

```diff
@@ -1792,6 +1792,16 @@ class Path extends NativeFieldWrapperClass2 {
   Path _transform(Float64List matrix4) native 'Path_transform';
 
   /// Computes the bounding rectangle for this path.
+  /// 
+  /// The returned bounds width and height may be larger or smaller than area
+  /// affected when Path is drawn.  For example, a path containing a straight
```

Comment:
I think we should drop the first sentence here. The area is correct in the example you give, it's just that it's not intuitive that the area for an axis-aligned path is zero, despite being correct.

Reply:
Actually, reading the Skia docs, the first sentence is right, it's just that the example you give isn't an example of that problem.

Reply:
In Skia, the path itself doesn't have a stroke width (that's a property of the paint used when the path is drawn).

Reply:
Yeah, the same is true for Flutter here - this is all adding up fine, but is worth documenting more clearly here.